src/Analyzers/CSharp/Tests/GenerateMethod/GenerateMethodTests.cs

```diff
@@ -11120,6 +11120,261 @@ private static int NewExpr()
             """);
     }
 
+    [Fact]
+    public async Task GenerateInCollection1()
+    {
+        await TestInRegularAndScriptAsync(
```

Comment:
No reason to change it, but if the only difference here was the type, a theory might have been handy to make it clear that's the only thing changing.